pkg/gitops/GitOpsConfigService.go

```diff
@@ -238,7 +239,12 @@ func (impl *GitOpsConfigServiceImpl) CreateGitOpsConfig(request *bean2.GitOpsCon
 		}
 	}
 	if strings.ToUpper(request.Provider) == GITHUB_PROVIDER {
-		request.Host = GITHUB_HOST + request.GitHubOrgId
+		hostUrl, err := url.Parse(request.Host)
```

Comment:
this code is twice may be we can extract it out

Reply:
done